disk-deactivate/disk-deactivate.jq

```diff
@@ -1,7 +1,7 @@
 # since lsblk lacks zfs support, we have to do it this way
 def remove:
   if .fstype == "zfs_member" then
-    "zpool destroy -f \(.label)"
+    "if type zpool >/dev/null; then zpool destroy -f \(.label); zpool labelclear -f \(.label); fi"
```

Comment:
So it looks like `zpool destroy` only works if the pool is mounted. `zpool labelclear` works in more cases.